### DISCUSSION TITLE
"Taskbar Clock Customization" : two lines and the second one "loading..."... ?

### DISCUSSION THREAD

**Discussion by sypqys** [ramensoftware/windhawk-mods#695] [Q&A] [answered]
Hello !

![3OOpvBPM5J](https://github.com/ramensoftware/windhawk-mods/assets/27967268/20f0cf1b-af42-414b-bbe6-3f5eca2e83a3)

I want only one line, not the line with loading. How I have to do for that ?

Thanks

**Comment by sypqys** [comment]
I would like to center the date...

Because I found how to delete the "loading..."

**Reply by m417z** [reply]
How did you hide it? The best way is to use the option that the mod provides, see https://github.com/ramensoftware/windhawk-mods/discussions/535#discussioncomment-8647444.

**Reply by sypqys** [reply]
Deleting "web%" content if I remember well...

**Comment by sypqys** [comment, marked as answer]
![t2gjyBwiav](https://github.com/ramensoftware/windhawk-mods/assets/27967268/98167323-ba97-42e1-8ca7-f5c888d0e2fe)
